charts/generic/pod-network-latency/pod-network-latency.chartserviceversion.yaml

```diff
@@ -2,16 +2,18 @@ apiVersion: litmuchaos.io/v1alpha1
 kind: ChartServiceVersion
 metadata:
   name: pod-network-latency
-  version: 0.1.2
+  version: 0.1.1
```

Comment:
Leave the version as it is. We  should not be downgrading a version.

Reply:
fixed this